docs/website/docs/dlt-ecosystem/destinations/sink.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Sink / Reverse ETL
+description: Sink function `dlt` destination for reverse ETL
+keywords: [reverse etl, sink, function]
+---
+
+# Sink function / Reverse ETL
+
+## Install dlt for Sink / reverse ETL
+** To install the DLT without additional dependencies **
+```
+pip install dlt
+```
+
+## Setup Guide
+### 1. Initialize the dlt project
+
+Let's start by initializing a new dlt project as follows:
+
+```bash
+dlt init chess sink
+```
+> 💡 This command will initialize your pipeline with chess as the source and Sink as the destination.
+
+The above command generates several files and directories, including `.dlt/secrets.toml`.
+
+### 2. Set up a sink function for your pipeline
+The sink destination differs from other destinations in that you do not need to provide connection credentials, but rather you provide a function which 
+gets called for all items loaded during a pipeline run or load operation. For the chess example, you can add the following lines at the top of the file.
+With the @dlt.sink decorator you can convert any function that takes two arguments into a dlt destination. 
+
+```python
+from dlt.common.typing import TDataItems
+from dlt.common.schema import TTableSchema
+
+@dlt.sink(batch_size=10)
+def sink(items: TDataItems, table: TTableSchema) -> None:
+    print(table["name"])
+    print(items)
+```
+
+To enable this sink destination in your chess example, replace the line `destination='sink'` with `destination=sink` (without the quotes) to directly reference
+the sink from your pipeline constructor. Now you can run your pipeline and see the output of all the items coming from the chess pipeline to your console.
+
+:::tip
+1. You can also remove the typing information (TDataItems and TTableSchema) from this example, typing generally are useful to know the shape of the incoming objects though.
+2. There are a few other ways for declaring sink functions for your pipeline described below.
+:::
+
+## Sink decorator function and signature
+
+The full signature of the sink decorator and a function is
+
+```python
+@dlt.sink(batch_size=10, loader_file_format="jsonl", name="my_sink")
+def sink(items: TDataItems, table: TTableSchema) -> None:
```

Comment:
Should we add a comment clarifying where users can import our internal types from?

Reply:
you can see that a couple of lines above this example